Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm

```diff
@@ -5265,6 +5270,8 @@ - (void)changeSelectionWithGestureAt:(CGPoint)point withGesture:(WKBEGestureType
 {
     logTextInteraction(__PRETTY_FUNCTION__, self.textInteractionLoupeGestureRecognizer, point, state, gestureType, std::nullopt, flags);
 
+    ALWAYS_LOG_WITH_STREAM(stream << "<MMG> changeSelectionWithGestureAt");
+
```

Comment:
Spurious logging.
```suggestion
```

Reply:
This and the below log are somehow not in my local branch, I'm not sure what happened.